app/components/app/app.js

```diff
@@ -127,6 +127,15 @@ var AppComponent = React.createClass({
     };
   },
 
+  initAppUtils: function(injectedUtils){
```

Comment:
What's the purpose of this? If it's for test mocking, there are better ways... Do a `git grep rewire` to see how [rewire](https://github.com/jhnns/rewire) is being used for dependency injection in tests.